openfasoc/generators/gdsfactory-gen/tapeout_and_RL/README.md

```diff
@@ -1,10 +1,29 @@
+# Installation and Getting Started
+These generators require python version >= 3.10  
+## Installing Layout Tools
+All layout dependencies can be installed with pip (the official python package manager). From here you can generate layouts with [Glayout](https://github.com/idea-fasoc/OpenFASOC/blob/main/openfasoc/generators/gdsfactory-gen/glayout/README.md#glayout).  
+
+```
+python3 -m pip install -r requirements.txt
+python3 -m pip install gdsfactory==7.7.0
+```
+**After installing python packages, you can test with:**  
+`python3 sky130_nist_tapeout.py gen_opamp --output_gds test1.gds`
+
+## Installing Simulation Tools
+A full install of these tools has only been verified on Linux platforms
+- [Skywater 130nm Open PDK](https://github.com/RTimothyEdwards/open_pdks)
```

Comment:
Is GF180 also supported? If yes, writing `Open PDKs` or `Skywater130 and GF180`  would be better.

Reply:
Gf180 is supported for generating opamps, but not for simulation